src/scvmm/HISTORY.rst

```diff
@@ -2,6 +2,8 @@
 
 Release History
 ===============
+1.2.0
+* Add command `az scvmm create-from-machines`.
```

Comment:
Why not consider designing the command as `az scvmm create --source Machines`? 
Perhaps in the future, it can support more different sources without creating new commands, which may have better scalability

Reply:
There was a small miss in my changes in `HISTORY.rst`: it's `az scvmm vm create-from-machines` insetad of `az scvmm create-from-machines`, have fixed it.

1. We wanted to maintain parity with the vmware commandlet - which is already published to the customers as `az connectedvmware vm create-from-machines`.
2. VM creation already has a standalone commandlet `az scvmm vm create` which takes in a bunch of parameters, for new vm creation from template. Wanted to have separate commandlets for creation from hybrid compute machine resoruces, to reduce customer confusion.

Reply:
OK, got it